src/json/Reader.cpp

```diff
@@ -0,0 +1,208 @@
+  std::string indent_;
+  if (!indent)
+    indent = &indent_;
+  switch (jv.kind()) {
+  case json::kind::object: {
+    os << "{\n";
+    indent->append(4, ' ');
+    auto const &obj = jv.get_object();
+    if (!obj.empty()) {
+      auto it = obj.begin();
+      for (;;) {
+        os << *indent << json::serialize(it->key()) << " : ";
+        pretty_print(os, it->value(), indent);
+        if (++it == obj.end())
+          break;
+        os << ",\n";
+      }
+    }
+    os << "\n";
+    indent->resize(indent->size() - 4);
+    os << *indent << "}";
+    break;
+  }
+
+  case json::kind::array: {
+    os << "[\n";
+    indent->append(4, ' ');
+    auto const &arr = jv.get_array();
+    if (!arr.empty()) {
+      auto it = arr.begin();
+      for (;;) {
+        os << *indent;
+        pretty_print(os, *it, indent);
+        if (++it == arr.end())
+          break;
+        os << ",\n";
+      }
+    }
+    os << "\n";
+    indent->resize(indent->size() - 4);
+    os << *indent << "]";
+    break;
+  }
+
+  case json::kind::string: {
+    os << json::serialize(jv.get_string());
+    break;
+  }
+
+  case json::kind::uint64:
+    os << jv.get_uint64();
+    break;
+
+  case json::kind::int64:
+    os << jv.get_int64();
+    break;
+
+  case json::kind::double_:
+    os << jv.get_double();
+    break;
+
+  case json::kind::bool_:
+    if (jv.get_bool())
+      os << "true";
+    else
+      os << "false";
+    break;
+
+  case json::kind::null:
+    os << "null";
+    break;
+  }
+
+  if (indent->empty())
+    os << "\n";
+}
+namespace Json {
+
+/**
+ * Get a standard string from json value if the type of this value is string,
+ * double, uint64 or int64 - otherwise the string is empty!
+ */
+[[gnu::pure]] 
+static std::string 
+GetValueAsString(const boost::json::value &value) {
+  std::string str;
+  char buffer_string[0x10];
+
+  switch (value.kind()) {
+  case boost::json::kind::string:
+    str = value.get_string();
+    break;
+  case boost::json::kind::uint64: {
+    std::snprintf(buffer_string, sizeof(buffer_string), "%" PRIu64,
+                  value.get_uint64());
+    str = buffer_string;
+  } break;
+  case boost::json::kind::int64: {
+    std::snprintf(buffer_string, sizeof(buffer_string), "%" PRId64,
+                  value.get_int64());
+    str = buffer_string;
+  } break;
+  case boost::json::kind::double_: {
+    std::snprintf(buffer_string, sizeof(buffer_string), "%f",
+                  value.get_double());
+    str = buffer_string;
+  } break;
+  case boost::json::kind::bool_:
+    str = value.get_bool() ? "true" : "false";
+    break;
+  case boost::json::kind::object:
+  case boost::json::kind::array:
+  default:
+    // in all this cases return an empty string
+    break;
+  }
+  return str;
+}
+
+class JsonReader {
+  boost::json::value json;
+
+public:
+ explicit JsonReader(const boost::json::standalone::value &_json)
+{
+    json = _json;
+    if (json.is_array())
+      json = json.get_array().at(0);
```

Comment:
Why?

Reply:
The one and only reason for this is to have a wrapper for the getting json from this server... I want to have a simple access to the string implementation of this data. Because this here is to much related to the real code/json string from the WeGlide server I have to move this to the Cloud/weglide path.
My hoped I can build a general component - but for now it is to much work for me (and maybe this isn't possible) - and so I have to skip it now :)

Reply:
The whole file is removed...